core/src/test/java/io/questdb/test/griffin/engine/functions/finance/FormatPriceFunctionFactoryTest.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2024 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.test.griffin.engine.functions.finance;
+
+import io.questdb.griffin.FunctionFactory;
+import io.questdb.griffin.engine.functions.finance.FormatPriceFunctionFactory;
+import io.questdb.test.griffin.engine.AbstractFunctionFactoryTest;
+import org.junit.Test;
+
+public class FormatPriceFunctionFactoryTest extends AbstractFunctionFactoryTest {
+
+    @Test
+    public void testFormatPriceHappyPath() throws Exception {
```

Comment:
could you add tests that produce multi-row result set? for example 

```sql
select format_price(rnd_double() * 100, 16) from long_sequence(50) order by 1
```

that would also sort the values of your function to a more complete exercise.